ocs_ci/ocs/acm/acm.py

```
@@ -849,8 +849,8 @@ def login_to_acm():
     driver = login_ui(url)
     page_nav = AcmPageNavigator()
     page_nav.page_has_loaded(retries=10, sleep_time=5)
-    locator = ["click-local-cluster", "click-admin-dropdown"]
-    expected_text = ["local-cluster", "Administrator"]
+    locator = ["click-local-cluster", "click-admin-dropdown", "click-fleet-management"]
+    expected_text = ["local-cluster", "Administrator", "Core platform"]
     for expected_text, locator in zip(expected_text, locator):
```

Ruff (0.15.21)

[warning] Loop control variable `expected_text` overrides iterable it iterates

(B020)